### DISCUSSION TITLE
cant get the lines or curves

### DISCUSSION THREAD

**Discussion by Eyderoe** [jsvine/pdfplumber#1118] [Ask for help with specific PDFs]
I want to extract the table ,but not as expected.
so I extract lines and curves ,but i cant get any.
might someone tell me what cause it
I will be appreciate if someone could explain

-----------------
import pdfplumber

raw = r".\ZUCK.pdf"
pdf = pdfplumber.open(raw)
page = pdf.pages[0]
print(page.lines)
print(page.curves)
[ZUCK.pdf](https://github.com/jsvine/pdfplumber/files/14887963/ZUCK.pdf)

**Comment by jsvine** [comment]
It appears that the linear graphical elements on that page have been captured as as `rects`, available via `page.rects`.

**Comment by Eyderoe** [comment]
thanks, it work well
